react/src/DealDetail.js

```diff
@@ -98,6 +106,9 @@ export function DealDetail(props) {
         console.error("parsing transfer params: "+e.message)
     }
 
+    const showRetryFailButtons = deal.Retry === 'manual' && deal.Error != ''
```

Comment:
Wouldn't we want to be able to retry `Auto` deals too, and not have to wait for a Boost reboot?

Reply:
Ah yeah that's a good point, let me make that change and test 👍